gdsfactory/geometry/union.py

```diff
@@ -65,9 +64,13 @@ def union(
             U.add_polygon(unioned_polygons, layer=layer)
     else:
         all_polygons = component.get_polygons(by_spec=False)
-        unioned_polygons = _union_polygons(
-            all_polygons,
-            precision=precision,
+        unioned_polygons = (
```

Comment:
Ditto here - this isn't listed in the PR description.

If this is a separate cleanup, IMO it should be a separate PR.

Reply:
same here,

this is some deprecated code that does nothing and remained since the gdstk to gdspy transition